SpringProject/src/main/java/com/gradproject2019/auth/data/LoginDto.java

```diff
@@ -0,0 +1,53 @@
+package com.gradproject2019.auth.data;
+
+import org.springframework.validation.annotation.Validated;
+
+import javax.validation.constraints.NotNull;
+
+@Validated
+public class LoginDto {
+    @NotNull(message = "Invalid entry in login field.")
```

Comment:
Please extract this message in a constant, no need to duplicate code.

Reply:
Would it be better to be more specific about which field is invalid? eg. `Invalid entry in username field`?